app/[locale]/client/settings/profile/billing/page.tsx

```
@@ -44,19 +46,18 @@
 						</div>
 						<h2 className="text-2xl font-bold text-gray-900 dark:text-gray-100 mb-3">
-							Welcome to Ever Works!
+							{t('WELCOME_TITLE')}
 						</h2>
 						<p className="text-gray-600 dark:text-gray-300 mb-8">
-							You&apos;re currently on the free plan. Upgrade to unlock premium features and start
-							building your success.
+							{t('FREE_PLAN_MESSAGE')}
 						</p>
 						<div className="space-y-3">
 							<button className="w-full bg-gradient-to-r from-theme-primary-600 to-theme-primary-600 text-white px-6 py-3 rounded-xl font-semibold hover:from-theme-primary-700 hover:to-theme-primary-700 transition-all duration-200 shadow-lg hover:shadow-xl transform hover:-translate-y-0.5">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)